OrbitGl/TimerTrack.cpp

```diff
@@ -63,7 +63,8 @@ void TimerTrack::UpdateBoxHeight() {
   box_height_ = time_graph_->GetLayout().GetTextBoxHeight();
 }
 
-void TimerTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick, PickingMode /*picking_mode*/) {
+void TimerTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick,
+                                  PickingMode /*picking_mode*/) {
```

Comment:
nit: you can drop /* */ it is clear from context that parameter is picking mode

Reply:
I'd prefer to leave it as we use this pattern a lot.